SamplifyAPIClient/Request.cs

```diff
@@ -29,12 +30,21 @@ internal async Task<APIResponse> Send(string host, HttpMethod method, string url
             string data = string.Empty;
             if (body != null)
             {
-                data = Util.Serialize(body);
+                data = Util.Serialize(body); 
                 msg.Content = new StringContent(data, System.Text.Encoding.UTF8, "application/json");
             }
             var res = await client.SendAsync(msg).ConfigureAwait(false);
-            string json = await res.Content.ReadAsStringAsync().ConfigureAwait(false);
+
             var reqID = this.GetHeaderValue(res.Headers, "x-request-id");
+
+            if (res.IsSuccessStatusCode && res.Content.Headers.ContentType.MediaType == "application/pdf")
```

Comment:
We need to restructure this part of code to keep all three conditions in sequence. Please text me on slack to discuss it.

Reply:
Fixed.